partition/roles/promtail/README.md

```diff
@@ -4,12 +4,10 @@ Deploys promtail in a systemd-managed Docker container.
 
 ## Variables
 
-| Name                              | Mandatory | Description                                |
-| --------------------------------- | --------- | ------------------------------------------ |
-| promtail_config_host_dir          |           | The location of the promtail config        |
-| promtail_image_name               | yes       | Image version of the promtail              |
-| promtail_image_tag                | yes       | Image tag of the promtail                  |
-| promtail_loki_push_endpoint       | yes       | The URL to the Loki push endpoint          |
-| promtail_scrape_configs           | yes       | A list containing the scrape configs       |
-| promtail_loki_basic_auth_username |           | The username for the client authentication |
-| promtail_loki_basic_auth_password |           | The password for the client authentication |
+| Name                     | Mandatory | Description                          |
+| ------------------------ | --------- | ------------------------------------ |
+| promtail_config_host_dir |           | The location of the promtail config  |
+| promtail_image_name      | yes       | Image version of the promtail        |
+| promtail_image_tag       | yes       | Image tag of the promtail            |
+| promtail_clients         | yes       | Configuration for promtail clients   |
```

Comment:
Add reference to https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write